content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -280,6 +280,25 @@ ensure that no two Services can collide. Kubernetes does that by allocating each
 Service its own IP address from within the `service-cluster-ip-range`
 CIDR range that is configured for the {{< glossary_tooltip term_id="kube-apiserver" text="API Server" >}}.
 
+#### Static Subrange NodePort (!!update later!!)
```

Comment:
Is this a draft? It looks like a draft.

Reply:
Yes, as far as the main content goes I have done/explained it correctly but some small re-wording and title changes would be great.